### ISSUE TITLE
Anything ever built with sehm?

### ISSUE THREAD

**Issue by Kinnardian** [hasenj/sehm#2]
Anything ever built with `sehm`?

**Comment by hasenj** [comment]
I don't think so. I never really built anything with it and doubt that anyone else did. If so I haven't heard of it.
